yarn.lock

```diff
@@ -6951,10 +6951,10 @@ graceful-fs@^4.1.4:
   integrity sha1-TK+tdrxi8C+gObL5Tpo906ORpyU=
 
 "graphiql@file:packages/graphiql":
-  version "0.17.0"
+  version "0.17.1"
   dependencies:
     codemirror "^5.47.0"
-    codemirror-graphql "^0.11.3"
+    codemirror-graphql "^0.11.4"
```

Comment:
huh, looks like we hadn't bumped our internal resolution on the last release somehow, i think this means the last release is missing a relatively obscure bugfix that doesnt impact graphiql in its suggested implementation, but may fix a bug for implementations that use that header